media_service/models.py

```diff
@@ -138,6 +138,10 @@ class Meta:
     def __unicode__(self):
         return u'{0}:{1}:{2}'.format(self.id, self.lti_context_id, self.title)
 
+
+def metadata_default():
```

Comment:
Would it be any more useful to make this a static method of a `Resource` object? Or a class/object property (that can be overridden)?